webui/react/src/constants/states.ts

```diff
@@ -64,31 +65,36 @@ export const terminalCommandStates: Set<CommandState> = new Set([
   CommandState.Terminating,
 ]);
 
-export const deletableRunStates: Set<CompoundRunState> = new Set([
+const runStateList = [
   RunState.Canceled,
   RunState.Completed,
-  RunState.Errored,
+  RunState.Error,
   RunState.DeleteFailed,
-]);
+] as const;
+
+export const deletableRunStates: Set<CompoundRunState> = new Set(runStateList);
 
 export const terminalRunStates: Set<CompoundRunState> = new Set([
   ...deletableRunStates,
   RunState.Deleted,
 ]);
 
-export const runStateToLabel: { [key in RunState]: string } = {
+export const terminalRunStatesKeys = [...runStateList, RunState.Deleted];
+
+export const runStateToLabel: { [key in RunStateValue]: string } = {
   [RunState.Active]: 'Active',
   [RunState.Running]: 'Running',
   [RunState.Canceled]: 'Canceled',
   [RunState.Completed]: 'Completed',
   [RunState.Deleted]: 'Deleted',
   [RunState.Deleting]: 'Deleting',
   [RunState.DeleteFailed]: 'Delete Failed',
-  [RunState.Errored]: 'Errored',
+  [RunState.Error]: 'Errored',
   [RunState.Paused]: 'Paused',
   [RunState.StoppingCanceled]: 'Canceling',
   [RunState.StoppingCompleted]: 'Completing',
   [RunState.StoppingError]: 'Erroring',
+  [RunState.StoppingKilled]: 'Killed',
```

Comment:
added this cuz it was mismatched with https://github.com/determined-ai/determined/blob/0.19.0-rc0/proto/src/determined/experiment/v1/experiment.proto#L12